kfs-purap/src/main/java/edu/arizona/kfs/module/purap/document/web/struts/PayeeSearchAction.java

```diff
@@ -0,0 +1,95 @@
+package edu.arizona.kfs.module.purap.document.web.struts;
```

Comment:
Why is this class in the kfs-purap project? Should be in package edu.arizona.kfs.tax.document.web.struts with the matching form.